### DISCUSSION TITLE
til/%EA%B0%9C%EB%B0%9C/algorithm/nqueen/

### DISCUSSION THREAD

**Discussion by giscus** [rlaisqls/blog-comments#12] [Announcements]
# til/%EA%B0%9C%EB%B0%9C/algorithm/nqueen/

보드판 크기를 N이라 두자.N을 6으로 나눈 나머지가 2 또는 3이 아니라면, 1부터 N까지의 수를 (짝수 오름차순) + (홀수 오름차순)으로 나열하도록 배치하면 된다.N을 6으로 나눈 나머지가 2이라면, 앞선 홀수 오름차순 리스트에서 1과 3의 위치를 바꾸고 5를 맨 뒤로 보낸다.  

https://blog.rlaisqls.site/til/%EA%B0%9C%EB%B0%9C/algorithm/nqueen/

**Comment by Yoochanhong** [comment]
이런 방법은 생각도 못했는데.. 신기하네요
